substrate-node/runtime/src/lib.rs

```diff
@@ -336,11 +336,20 @@ impl ChangeNode for NodeChanged {
     }
 }
 
+parameter_types! {
+    pub const MaxTwinIpLength: u32 = 37;
```

Comment:
Max IP length is 39 (if we exclude special 6 to 4 addresses which we reject anyway).

Also, this should be a constant in the correct pallet, as the lenght of an IPv6 addresses is a global constant, not something specialized for the runtime